app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -4,21 +4,57 @@
 
 package org.mozilla.focus.home.pocket
 
+import android.graphics.Color
 import android.os.Bundle
 import android.support.v4.app.Fragment
+import android.support.v7.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import kotlinx.android.synthetic.main.fragment_pocket_video.view.*
 import org.mozilla.focus.R
 
 /** A feed of Pocket videos. */
 class PocketVideoFragment : Fragment() {
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        return inflater.inflate(R.layout.fragment_pocket_video, container, false)
+        val layout = inflater.inflate(R.layout.fragment_pocket_video, container, false)
+        layout.videoFeed.gridView.adapter = PocketVideoAdapter()
+        return layout
     }
 
     companion object {
         fun create() = PocketVideoFragment()
     }
 }
+
+private class PocketVideo(
+        val title: String,
+        val url: String,
+        val imageUrl: String
+)
+
+private class PocketVideoAdapter : RecyclerView.Adapter<PocketVideoViewHolder>() {
+
+    private val feedItems = List(8) { PocketVideo("Mirror-Polished Japanese $it Ball Challenge Crushed", "youtube.com/tv", "youtube.com/tv") }
+
+    override fun getItemCount() = feedItems.size
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
+            PocketVideoViewHolder(LayoutInflater.from(parent.context).inflate(R.layout.pocket_video_item, parent, false))
```

Comment:
Oh, so making this the "theater mode" is as simple as using CardViews for items, rather than making custom layouts?